lib/Segments/DynamicSegments/Filters/EmailAction.php

```diff
@@ -75,36 +90,45 @@ public function apply(QueryBuilder $queryBuilder, DynamicSegmentFilterEntity $fi
         'statssent',
         $statsTable,
         'stats',
-        $this->createNotStatsJoinCondition($filter, $action, $linkId, $parameterSuffix)
+        $this->createNotStatsJoinCondition($action, $parameterSuffix, $linkId)
       )->setParameter('newsletter' . $parameterSuffix, $newsletterId);
       $where .= ' AND stats.id IS NULL';
-    } else if ($action === self::ACTION_CLICKED_ANY) {
-      $excludedLinks = [
-        '[link:subscription_unsubscribe_url]',
-        '[link:subscription_instant_unsubscribe_url]',
-        '[link:newsletter_view_in_browser_url]',
-        '[link:subscription_manage_url]',
-      ];
+    } elseif (($action === EmailAction::ACTION_OPENED) && ($operator === DynamicSegmentFilterData::OPERATOR_NONE)) {
```

Comment:
When I select more than one newsletter with the "none of" option, I get an SQL error while counting the number of subscribers:
> An exception occurred while executing 'SELECT IFNULL(SUM(            CASE WHEN wpdev_mailpoet_subscribers.deleted_at IS NULL              THEN 1 ELSE 0 END        ), 0) as `all`, IFNULL(SUM(            CASE WHEN wpdev_mailpoet_subscribers.deleted_at IS NOT NULL              THEN 1 ELSE 0 END        ), 0) as trash, IFNULL(SUM(            CASE WHEN wpdev_mailpoet_subscribers.status = ? AND wpdev_mailpoet_subscribers.deleted_at IS NULL              THEN 1 ELSE 0 END        ), 0) as ?, IFNULL(SUM(          CASE WHEN wpdev_mailpoet_subscribers.status = ? AND wpdev_mailpoet_subscribers.deleted_at IS NULL            THEN 1 ELSE 0 END        ), 0) as ?, IFNULL(SUM(          CASE WHEN wpdev_mailpoet_subscribers.status = ? AND wpdev_mailpoet_subscribers.deleted_at IS NULL            THEN 1 ELSE 0 END        ), 0) as ?, IFNULL(SUM(          CASE WHEN wpdev_mailpoet_subscribers.status = ? AND wpdev_mailpoet_subscribers.deleted_at IS NULL            THEN 1 ELSE 0 END        ), 0) as ?, IFNULL(SUM(          CASE WHEN wpdev_mailpoet_subscribers.status = ? AND wpdev_mailpoet_subscribers.deleted_at IS NULL            THEN 1 ELSE 0 END        ), 0) as ? FROM wpdev_mailpoet_subscribers INNER JOIN (SELECT DISTINCT wpdev_mailpoet_subscribers.id as inner_subscriber_id FROM wpdev_mailpoet_subscribers INNER JOIN wpdev_mailpoet_statistics_newsletters statssent ON wpdev_mailpoet_subscribers.id = statssent.subscriber_id AND statssent.newsletter_id IN (?, ?, ?) LEFT JOIN wpdev_mailpoet_statistics_opens stats ON statssent.subscriber_id = stats.subscriber_id AND stats.newsletter_id = (?, ?, ?) WHERE 1 AND stats.id IS NULL) a0 ON a0.inner_subscriber_id = wpdev_mailpoet_subscribers.id WHERE wpdev_mailpoet_subscribers.deleted_at IS NULL' with params ["subscribed", "subscribed", "unsubscribed", "unsubscribed", "inactive", "inactive", "unconfirmed", "unconfirmed", "bounced", "bounced", 18, 10, 7, 18, 10, 7]:SQLSTATE[21000]: Cardinality violation: 1241 Operand should contain 1 column(s)